testing/opae-cxx/test_events_cxx_core.cpp

```diff
@@ -102,6 +105,7 @@ class events_cxx_core : public ::testing::TestWithParam<std::string> {
     }
   }
 
+
   handle::ptr_t handle_;
```

Comment:
Need to move the declaration of tokens_ to the top to match the initialization order in the constructor.